components/Meta.js

```diff
@@ -11,7 +11,7 @@ const Meta = props => (
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta charSet="utf-8" />
     <link rel="shortcut icon" type="image/x-icon" href="/images/favicon.png" />
-    <link href="https://fonts.googleapis.com/css?family=Domine:400,700|Roboto:400,700&display=swap" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400&display=swap" rel="stylesheet" />
```

Comment:
> Love these graphics! This page is looking great.
> 
> Did you look into using Next.js's built-in [static file serving](https://nextjs.org/docs#static-file-serving-eg-images)?

@nholden I think this method is using the Next.js built-in static file serving. Can you double check for me in case I am missing some part of that?

Reply:
The [documentation](https://nextjs.org/docs#static-file-serving-eg-images) makes it look like static file serving happens in Next.js only for files living under the `public` directory. Is there something different with our environment from the Next.js docs?